### PR TITLE
fix(editor):  Shorten overflowing Node Label in InputLabels on hover and focus

### PR DESCRIPTION
**Pull request by CharlieKolb**
## Summary

Shorten the label on InputLabel on hover and on focus if options are in use.

Before:
Screenshot 2024-10-04 at 16 38 03

After:
Screenshot 2024-10-04 at 16 39 38

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/ADO-2626/display-name-of-ndv-parameters-should-wrap-and-not-overlap-with

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/design-system/src/components/N8nInputLabel/InputLabel.vue [hunk 1]
```diff
@@ -43,11 +43,18 @@ const addTargetBlank = (html: string) =>
 				[$style.heading]: !!label,
 				[$style.underline]: underline,
 				[$style[size]]: true,
-				[$style.overflow]: !!$slots.options,
 			}"
 		>
 			<div v-if="label" :class="$style.title">
-				<N8nText :bold="bold" :size="size" :compact="compact" :color="color">
+				<N8nText
```
Comment (mutdmour) [inline packages/design-system/src/components/N8nInputLabel/InputLabel.vue:49]: could there be some unit tests added here?

File: packages/design-system/src/components/N8nInputLabel/InputLabel.vue [hunk 2]
```diff
@@ -43,11 +43,18 @@ const addTargetBlank = (html: string) =>
 				[$style.heading]: !!label,
 				[$style.underline]: underline,
 				[$style[size]]: true,
-				[$style.overflow]: !!$slots.options,
```
[the remark this answers is not in the document]
Comment (CharlieKolb) [reply packages/design-system/src/components/N8nInputLabel/InputLabel.vue:46]: Discussed offline, we're committing to changing all InputLabels instead, will update PR description

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.63.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.63.0)

**Comment by janober** [comment]
Got released with [`n8n@1.63.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.63.0)

**Pull request merged**